docs/operating-modes.md

```diff
@@ -14,17 +14,36 @@ You will also be able to run Stryker from your solution context in the future.
 
 ## Test project context
 
-Just run `dotnet stryker` in your test project folder.  
-Stryker will automatically detect the source project based on the project reference. If your test project has multiple project references, you need to specify the source project with `--project <filename>` ([docs here](https://stryker-mutator.io/docs/stryker-net/configuration/#project-file-name)). You can only specify one source project at the time.
+Run `dotnet stryker` in your test project folder.
+
+```bash
+cd C:\myTestProjectDir
```

Comment:
We should give full example paths in all the examples so the difference in location between all examples is obvious